sympy/ntheory/factor_.py

```diff
@@ -1518,12 +1518,23 @@ def factorrat(rat, limit=None, use_trial=True, use_rho=True, use_pm1=True,
 
 
 
-def primefactors(n, limit=None, verbose=False):
+def primefactors(n, **kwargs):
```

Comment:
For people who already use this with positional args, I think we need to leave those in place and add `**kwargs` to the end. Then `kwargs.update(dict(limit=limit, verbose=verbose))` before the call to `factorint`.

Reply:
I did not give it enough consideration.